redaxo/src/addons/mediapool/lib/mediapool.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+class rex_mediapool
```

Comment:
Hier fehlt die `@package`-Angabe, und ich würde es final machen.

```suggestion
 /**
  * @package redaxo\mediapool
  */
final class rex_mediapool
```